### ISSUE TITLE
[feature request] Page movement

### ISSUE THREAD

**Issue by folofjc** [Wojnach/vimflowy#21]
This is a great extension! One of the things I am missing (which would be very helpful with long lists) is some of the cursor movements:

- `H`: move cursor to item at the top of the "page"
- `M`: put the cursor in the middle of the "page"
- `L`: Put cursor on item at the bottom of the "page"

In the above, the "page" is the things that you can currently see on the page. Not sure if that is feasible.

- `zz`: Whatever item is under the current cursor position, scroll the page so that it is in the middle
- `zt`: Whatever item is under the cursor, scroll the page so that it is at the top
- `zb`: Whatever item is under the cursor, scroll the page so that it is at the bottom

Obviously, for these it depends on how much can be scrolled.

**Comment by Wojnach** [comment]
yes, I've been missing zz too. All of them should be easy to implement. zt won't always work though due to how the workflowy page works. I'll have a look

**Comment by folofjc** [comment]
Thanks!

For the `zz` one, even if there was a mapping to scroll the window without the arrow keys.

**Comment by awillats** [comment]
Hey all, just implemented zt, zz, zb in a local branch! Is there any interest in working on a merge request to get this into master? 

It ended up being relatively straightforward but has a couple of hacks:
1.) I ended up throwing the keybinds in transparentKeybinding.js rather than keybindings.js. For some reason I couldn't get them to trigger in keybindings - maybe because they're a pair/sequence rather than a single key?
2.) Perhaps as a result of them being transparent, they would sometimes leak character inserts through, so I had to add a SimulateEscapeNormalMode to reset the normal mode state.
